templates/tab/js/m365/package.json

```diff
@@ -9,7 +9,7 @@
     "@microsoft/mgt-teamsfx-provider": "2.4.1-next.teamsfx.1552a26",
     "@microsoft/microsoft-graph-client": "^3.0.1",
     "@microsoft/teams-js": "2.0.0-beta.6",
-    "@microsoft/teamsfx": "0.7.0-beta.0",
+    "@microsoft/teamsfx": "1.0.0-rc.0",
```

Comment:
No need to update `teamsfx` version to `1.0.0-rc.0` for M365 templates. Once `teams-js@2.0.0` is released, we will release `teamsfx@2.0.0-beta.0` manually, and then update the `teams-js` and `teamsfx` version.

Reply:
sure, revert this change